tools/meson-check-help.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/sh -eu
 
+export SYSTEMD_LOG_LEVEL=info
```

Comment:
iirc this is a bashism, no? we either should write this as 

```
SYSTEMD_LOG_LEVEL=info
export SYSTEMD_LOG_LEVEL
```

or update the shebang line?

Reply:
https://unix.stackexchange.com/questions/193095/where-is-export-var-value-not-available
> It (`export var=22`) is not a bashism but a POSIX compliant syntax. It actually started as a kshism quite a long time ago and was later adopted by almost all Bourne syntax based shells. The only notorious exception is /bin/sh on Solaris 10 and older which sticks to the legacy Bourne shell syntax. Hopefully, Solaris 11 uses a POSIX compliant shell as /bin/sh.

But I do not know whether this is true or not. Where can I find the solid the POSIX compliant syntax?

Reply:
I think that's good enough, sounds like it's supported everywhere.